armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -18,6 +18,13 @@
 from armi import runLog
 from armi.reactor.flags import Flags
 
+TARGET_FLAGS_IN_PREFERRED_ORDER = [
+    Flags.FUEL,
+    Flags.CONTROL,
+    Flags.POISON,
+    Flags.SHIELD,
```

Comment:
@opotowsky @onufer Does it make sense to include `Flags.SLUG` in this list as well? Possibly the last value in the list.